src/lsm/manifest_level_fuzz.zig

```diff
@@ -0,0 +1,426 @@
+//! Fuzz ManifestLevel insert_table()/set_snapshot_max()/remove_table*()/iterator().
+//!
+//! Invariants checked:
+//! 
+//! - Inserted tables become visible (and invisible, given the snapshot) from the iterator.
+//! - Tables with updated snapshot_max become invisible to future snapshots from the iterator.
+//! - Tables visible to the current/future snapshot can be removed.
+//! - Tables invisible to the current/future/past snapshots can be removed.
+//! - Tables can be inserted, updated, and removed in any order.
```

Comment:
```
+ All tables can be updated.
+ All tables can be removed.
```

Reply:
We're still missing `Strategy` here, a short sentence or paragraph to tell the story of how the fuzzer maximizes coverage-to-test-code ratio.